Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.yml

```diff
@@ -0,0 +1,1208 @@
+    - contextPath: SekoiaXDR.Events.Query.expiration_date
+      description: The date on which the event will expire.
+    - contextPath: SekoiaXDR.Events.Query.created_at
+      description: The time at which the event was created.
+    - contextPath: SekoiaXDR.Events.Query.status
+      description: The status of the event.
+    - contextPath: SekoiaXDR.Events.Query.view_uuid
+      description: The unique identifier for the view associated with the event.
+    - contextPath: SekoiaXDR.Events.Query.canceled_at
+      description: The time at which the event was canceled.
+    - contextPath: SekoiaXDR.Events.Query.only_eternal
+      description: A boolean indicating whether the event is only eternal.
+    - contextPath: SekoiaXDR.Events.Query.results_ttl
+      description: The time-to-live for the event results.
+    - contextPath: SekoiaXDR.Events.Query.canceled_by
+      description: The user who canceled the event.
+    - contextPath: SekoiaXDR.Events.Query.term_lang
+      description: The language of the search term associated with the event.
+    - contextPath: SekoiaXDR.Events.Query.ended_at
+      description: The time at which the event ended.
+    - contextPath: SekoiaXDR.Events.Query.earliest_time
+      description: The earliest time associated with the event.
+    - contextPath: SekoiaXDR.Events.Query.max_last_events
+      description: The maximum number of events to include in the results.
+    - contextPath: SekoiaXDR.Events.Query.canceled_by_type
+      description: The type of the user who canceled the event.
+    - contextPath: SekoiaXDR.Events.Query.total
+      description: The total number of events associated with the event.
+    - contextPath: SekoiaXDR.Events.Query.created_by_type
+      description: The type of the user who created the event.
+    - contextPath: SekoiaXDR.Events.Query.community_uuids
+      description: The list of community UUIDs associated with the event.
+    - contextPath: SekoiaXDR.Events.Query.filters.field
+      description: The field associated with the filter.
+    - contextPath: SekoiaXDR.Events.Query.filters.value
+      description: The value associated with the filter.
+    - contextPath: SekoiaXDR.Events.Query.filters.operator
+      description: The operator used in the filter.
+    - contextPath: SekoiaXDR.Events.Query.filters.excluded
+      description: Indicates whether the filter is excluded or not.
+    - contextPath: SekoiaXDR.Events.Query.filters.disabled
+      description: Indicates whether the filter is disabled or not.
+    description: Command to create an event search job on Sekoia XDR, after this execute "sekoia-xdr-status-events-query" to see the status of the query job and "sekoia-xdr-results-events-query" to retrieve the results..
+  - name: sekoia-xdr-events-status-query
+    arguments:
+    - name: uuid
+      required: true
+      description: UUID of the query executed previously with the "sekoia-xdr-query-events" command.
```

Comment:
The same as above. It is recommended to add to the argument description how to retrieve the UUID. To this argument, and to the rest UUID arguments.